src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml.cs

```diff
@@ -566,10 +578,27 @@ private void updateLibraryZoomScaleValueLabel(Slider slider)
             //The value is decreased to 480 because the margin begins at - 480
             //This is the relation between the margin in pixels and the value of the percentage
             double marginValue = (790 * percentage / 275) - 480;
-            if (lblZoomScalingValue != null)
+            if (label != null)
+            {
+                label.Margin = new Thickness(marginValue, 0, 0, 0);
+                label.Content = slider.Value.ToString() + "%";
```

Comment:
We can make this "%" as a string constant in the class and use it, to avoid creation of multiple string literals.

Reply:
hi @reddyashish I thought about that, but I am not sure if that would be accurate for multiple languages, e.g. French will force `15%` to be `15 %`, I feel this cant be achieved unless we make this whole string a resource so localization can replace it

Reply:
Sounds good.